packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -1326,6 +1326,29 @@ void main() {
     }
     expect(items.take(8), orderedEquals(<int>[0, 1, 2, 3, 4, 5, 6, 7]));
   });
+
+  testWidgets('ReorderableListView thows an error when key is not passed to its children',(WidgetTester tester) async {
```

Comment:
Actually this is testing the wrong thing. The default constructor for `ReorderableListView` already checks all the children for a key and asserts if one doesn't have it. (This is why a lot of the tests jobs are failing).

Instead the you need to test the item builder variant constructor:

```dart
    testWidgets('ReorderableListView throws an error when key is not passed to its children', (WidgetTester tester) async {
      final Widget reorderableListView = ReorderableListView.builder(
        itemBuilder: (BuildContext context, int index) {
          return SizedBox(child: Text('Item $index'));
        },
        itemCount: 3,
        onReorder: (int oldIndex, int newIndex) { },
      );
      await tester.pumpWidget(MaterialApp(
        home: reorderableListView,
      ));
      final dynamic exception = tester.takeException();
      expect(exception, isFlutterError);
      expect(exception.toString(), contains('Every item of ReorderableListView must have a key.'));
    });
```

Reply:
Thanks @darrenaustin